README.rst

```diff
@@ -23,15 +23,13 @@ For new users, I also recommend reading `How to ask for help <https://pangeo-xes
 .. _xarray: http://xarray.pydata.org
 .. _dask: https://dask.org/
 
-.. |pypi| image:: https://badge.fury.io/py/xesmf.svg
-   :target: https://badge.fury.io/py/xesmf
-   :alt: pypi package
+.. |conda| image:: https://anaconda.org/conda-forge/xesmf/badges/version.svg
+   :target: https://anaconda.org/conda-forge/xesmf/
```

Comment:
And with an ``:alt:`` entry?

Reply:
I'm not familiar with that. welcome to push to my branch.